types/aws-lambda/index.d.ts

```diff
@@ -190,6 +197,7 @@ interface ProxyResult {
  */
 export type Handler = (event: any, context: Context, callback?: Callback) => void;
 export type ProxyHandler = (event: APIGatewayEvent, context: Context, callback?: ProxyCallback) => void;
+export type CustomAuthorizerHandler = (event: CustomAuthorizerEvent, context: Context, callback?: Callback) => void;
```

Comment:
@buggy 

I think your opinion is correct.

I will make corrections as you have indicated.

Thank you.